packages/mds-api-server/tests/index.spec.ts

```diff
@@ -44,163 +43,133 @@ const request = supertest(api)
+      status: expect.stringContaining('Running')
+    })
   })
 
-  it('verifies health (MAINTENANCE)', done => {
+  it('verifies health (MAINTENANCE)', async () => {
     process.env.MAINTENANCE = 'Testing'
-    request
-      .get('/health')
-      .expect(HttpStatus.OK)
-      .end((err, result) => {
-        test.value(result).hasHeader('content-type', APP_JSON)
-        test.object(result.body).hasProperty('name')
-        test.object(result.body).hasProperty('version')
-        test.object(result.body).hasProperty('node')
-        test.object(result.body).hasProperty('build')
-        test.object(result.body).hasProperty('process')
-        test.object(result.body).hasProperty('memory')
-        test.object(result.body).hasProperty('uptime')
-        test.object(result.body).hasProperty('status', 'Testing (MAINTENANCE)')
-        done(err)
-      })
+    const result = await request.get('/health').expect(HttpStatus.OK)
+
+    expect(result.headers['content-type']).toStrictEqual(APP_JSON)
+    expect(result.body).toMatchObject({
+      name: expect.any(String),
+      version: expect.any(String),
+      node: expect.any(String),
+      build: expect.any(Object),
+      process: expect.any(Number),
+      memory: expect.any(Object),
+      uptime: expect.any(Number),
+      status: expect.stringContaining('Testing (MAINTENANCE)')
+    })
   })
 
-  it('verifies MAINTENANCE repsonse', done => {
+  it('verifies MAINTENANCE repsonse', async () => {
     process.env.MAINTENANCE = 'Testing'
-    request
+    const result = await request
       .get('/this-is-a-bad-route-but-it-should-return-503-in-maintenance-mode')
       .expect(HttpStatus.SERVICE_UNAVAILABLE)
-      .end((err, result) => {
-        test.value(result).hasHeader('content-type', APP_JSON)
-        test.object(result.body).hasProperty('status', 'Testing (MAINTENANCE)')
-        done(err)
-      })
+
+    expect(result.headers['content-type']).toStrictEqual(APP_JSON)
+    expect(result.body.status).toStrictEqual('Testing (MAINTENANCE)')
   })
 
-  it('verifies MAINTENANCE passthrough', done => {
-    request
-      .get('/this-is-a-bad-route-so-it-should-normally-return-404')
-      .expect(HttpStatus.NOT_FOUND)
-      .end(err => {
-        done(err)
-      })
+  it('verifies MAINTENANCE passthrough', async () => {
+    await request.get('/this-is-a-bad-route-so-it-should-normally-return-404').expect(HttpStatus.NOT_FOUND)
   })
 
-  it('verifies keepAliveTimeout setting', done => {
-    let error
+  it('verifies keepAliveTimeout setting', async () => {
     process.env.HTTP_KEEP_ALIVE_TIMEOUT = '3000'
     const server = HttpServer(api)
     try {
-      test.value(server.keepAliveTimeout).is(Number(process.env.HTTP_KEEP_ALIVE_TIMEOUT))
-    } catch (err) {
-      error = err
+      expect(server.keepAliveTimeout).toStrictEqual(Number(process.env.HTTP_KEEP_ALIVE_TIMEOUT))
+    } finally {
+      server.close()
     }
-    server.close()
-    done(error)
```

Comment:
I have no idea why this needed to be a try/catch to begin with, or a try/finally now. Could this just not have any `try` wrapping?